test/test_standard_pipelines.py

```diff
@@ -0,0 +1,326 @@
+from pathlib import Path
+
+import os
+import math
+import pytest
+
+from haystack.document_stores.elasticsearch import ElasticsearchDocumentStore
+from haystack.pipelines import (
+    Pipeline,
+    FAQPipeline,
+    DocumentSearchPipeline,
+    RootNode,
+    MostSimilarDocumentsPipeline,
+)
+from haystack.nodes import DensePassageRetriever,  ElasticsearchRetriever, SklearnQueryClassifier, TransformersQueryClassifier, JoinDocuments
+from haystack.schema import Document
+
+
+@pytest.mark.parametrize(
+    "retriever,document_store",
+    [
+        ("embedding", "memory"),
+        ("embedding", "faiss"),
+        ("embedding", "milvus"),
+        ("embedding", "elasticsearch"),
+    ],
+    indirect=True,
+)
+def test_faq_pipeline(retriever, document_store):
+    documents = [
+        {
+            "content": "How to test module-1?",
+            "meta": {"source": "wiki1", "answer": "Using tests for module-1"},
+        },
+        {
+            "content": "How to test module-2?",
+            "meta": {"source": "wiki2", "answer": "Using tests for module-2"},
+        },
+        {
+            "content": "How to test module-3?",
+            "meta": {"source": "wiki3", "answer": "Using tests for module-3"},
+        },
+        {
+            "content": "How to test module-4?",
+            "meta": {"source": "wiki4", "answer": "Using tests for module-4"},
+        },
+        {
+            "content": "How to test module-5?",
+            "meta": {"source": "wiki5", "answer": "Using tests for module-5"},
+        },
+    ]
+
+    document_store.write_documents(documents)
+    document_store.update_embeddings(retriever)
+
+    pipeline = FAQPipeline(retriever=retriever)
+
+    output = pipeline.run(query="How to test this?", params={"Retriever": {"top_k": 3}})
+    assert len(output["answers"]) == 3
+    assert output["query"].startswith("How to")
+    assert output["answers"][0].answer.startswith("Using tests")
+
+    if isinstance(document_store, ElasticsearchDocumentStore):
+        output = pipeline.run(query="How to test this?", params={"Retriever": {"filters": {"source": ["wiki2"]}, "top_k": 5}})
+        assert len(output["answers"]) == 1
+
+
+@pytest.mark.parametrize("retriever_with_docs", ["embedding"], indirect=True)
+def test_document_search_pipeline(retriever, document_store):
```

Comment:
we have two tests with the same name here (also had that before but let's fix that now)